### DISCUSSION TITLE
Relative Humidity on Initial Region

### DISCUSSION THREAD

**Discussion by giaschweitzer** [firemodels/fds#13778] [Q&A]
Per the FDS User Guide, relative humidity can be specified on MISC to set the ambient relative humidity (and the default relative humidity is 40%). For my case, I want to specify ambient conditions and different initial conditions.

If I wanted to use an Init Region to specify a starting temperature and relative humidity that is different than the specified ambient conditions, how would I do this? I know how I can specify temperature. But how can I do relative humidity? Can it be specified as a percentage? Or only a mass fraction?

For example, ambient conditions would be Temperature = 35.61C and RH=72.7% and initial region conditions would be Temperature=23.9C and RH=44%.

**Comment by drjfloyd** [comment]
You could create a simple case where you set TMPA=23.9 and HUMIDITY=44 and look at the .out file to get the values for all the initial species to use for the INIT.

**Reply by giaschweitzer** [reply]
Thank you!

As a follow-up question, is it possible to have two vents on an exterior mesh boundary that overlap but are activated at different times? The goal is to supply air at T1 from a vent for a specified period of time, then switch over to supply air T2.

**Reply by drjfloyd** [reply]
7.4.1 Basics
Vents are specified in a similar manner to obstructions, with the sextuplet XB denoting a plane abutting a
solid surface. Two of the six coordinates must be the same, denoting a plane as opposed to a solid. Note that
only one VENT may be specified for any given solid surface. If VENTs overlap , FDS will output a warning
message and ignore redundant VENT lines.
